spec/defines/router_db_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'spec_helper'
+describe 'rancid::router_db' do
+  let(:title) { 'group1' }
+
+  let(:facts) { { :osfamily => 'Debian' } }
+  let(:pre_condition) {
+    'class rancid {
```

Comment:
Should not need a pre-condition to set rancid variables.  Really should not need the `include rancid` as the defined type should have that in it if it is necessary.

Reply:
Except that in the real world, `router_db` is only intended to be invoked _from_ class rancid, and hence it is implicitly included.  Here, in unit tests, that's not the case, so I'm taking steps to setup the environment that `router_db` expects.

Reply:
Just add `include rancid` to the defined type. Its absence is not by design.

Reply:
Now that you have `include rancid` in the defined type, you should not need a `pre_condition` at all.